src/heroes/sombra/translocator_cooldown_reset.opy

```diff
@@ -0,0 +1,49 @@
+#Global variables
+
+globalvar STEALTH_REFUND_FRACTION 0
+globalvar NEGATIVE_STATUS_PREVENTS_REFUND 1
+globalvar MINIMUM_TRANSLOCATOR_COOLDOWN 2
+
+
+#Player variables
+
+playervar stealth_timer 0
+playervar stealth_max_Cooldown 1
+playervar ult_charge_Percent_cache 2
+
+
+rule "[sombra/translocator_cooldown_reset.opy]: Sombra stealth refund variables":
+    STEALTH_REFUND_FRACTION = createWorkshopSetting(int[0:100], "Gameplay Settings", "Percent of Stealth Timer Remaining Refunded Towards Translocator Cooldown", 40, 0) / 100
+    NEGATIVE_STATUS_PREVENTS_REFUND = createWorkshopSetting(bool, "Gameplay Settings", "Negative Status Effects Cancelling Stealth Prevents Translocator Cooldown Refund", true, 1)
+    MINIMUM_TRANSLOCATOR_COOLDOWN = 4
+    
+    
+rule "[sombra/translocator_cooldown_reset.opy]: When Translocator is used, begin tracking how long Sombra is in stealth":
+    @Event eachPlayer
+    @Condition (eventPlayer.getCurrentHero() == Hero.SOMBRA) == true
+    @Condition eventPlayer.isUsingAbility2() == true
+    
+    eventPlayer.stealth_timer = 5.5
+    waitUntil(eventPlayer.isUsingAbility2() == false, 1000000)
+    chase(eventPlayer.stealth_timer, 0, rate=1, ChaseReeval.NONE)
+    #Sombra can't break stealth during the cast time of stealth
+    wait(0.5, Wait.RESTART_WHEN_TRUE)
```

Comment:
Move balance numbers to `adj_constants.opy` and ow numbers to `ow2_constants.opy`